lib/mina/infinum/ecs/params.rb

```diff
@@ -21,7 +21,9 @@
     are fetched.
   TXT
```

Comment:
Let's update the documentation

```suggestion
desc <<~TXT
   Save AWS Param Store params to a file
   
   Params are by default stored to .env file, you can override
   the location with :path. For example:
   $ mina params:pull path=.env.local

   See params:read documentation for details on how params
   are fetched.
 TXT
```